Makefile

```
@@ -75,6 +75,13 @@ testSuite:
 	@echo "testSuite"
 	./tools/check/check_testSuite.sh
 
+check-parallel:
+	# Make sure no tests are run in parallel to prevent possible unstable tests.
+    # See https://github.com/pingcap/tidb/pull/30692.
+	@! find . -name "*_test.go" -not -path "./vendor/*" -print0 | \
+      xargs -0 grep -E -n "t.Parallel()" || \
```

```suggestion
      xargs -0 grep -F -n "t.Parallel()" || \
```

`.` and `()` are special characters in egrep...